src/modules/GameHelper.ts

```diff
@@ -2,6 +2,8 @@ import {
     Observable as KnockoutObservable,
     Computed as KnockoutComputed,
 } from 'knockout';
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+import * as ko from 'knockout';
```

Comment:
I'm pretty sure this causes any observables created with this ko to not actually trigger any UI updates or computeds made with the global ko - easily tested with the questlist refresh tooltip which uses GameHelper.formatterLetterTimeUntilTomorrow

Last time I had this issue I think I decided it was because this ko is a different one to the one we use when applying bindings? That one coming from the knockout script tag included in the html. I did have an idea though which would save us from having ko as a global in modules - we could remove that script tag and instead temporaryWindowInject whatever `ko` modules is using?

---

Just tried that out for a few minutes and it looks like it isn't as simple as I thought...

Seems like the best way to solve this for now would be to add ko as a global in the jest.config.js and remove this import I am commenting on
```
  globals: {
    ko: require('knockout'),
  },
```

Reply:
I cannot figure out how to get this to work, the globals route just fails to run the tests for me with:
`ko.observable is not a function`

Reply:
can you push up a separate branch for me to see what you are doing differently? It was working for me when I tried it

Reply:
Added here: https://github.com/pokeclicker/pokeclicker/tree/sparr0w/testing-jest-ko
Just removed the import line, added the globals.

Reply:
Keep the `import * as ko from 'knockout'` in the modules.test.ts, just remove the one from GameHelper.
The one in test files should be fine since it doesn't affect the actual game code

Reply:
Or it might be possible to get the jest global one in your tests with `globalThis.ko.observable()`

Reply:
Gives me the same problem if it remains in `modules.test.ts` or not

Reply:
Gonna make a PR with that branch, see if it fails with the GitHub tests

Reply:
Passes fine on there too 🤔 

Gonna try another clean install and see if that fixed it on my end.

Reply:
Tried clean install, and on another computer which hadn't run any of the tests yet, same error, might be Windows related.

Reply:
Got it to work with `maxWorkers: '1',` not entirely sure why though.
Only thing I could find was this: https://github.com/facebook/jest/issues/11956#issuecomment-1458178986